<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="http://www.springframework.org/schema/beans"
       xmlns:context="http://www.springframework.org/schema/context"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xsi:schemaLocation="http://www.springframework.org/schema/beans
       http://www.springframework.org/schema/beans/spring-beans.xsd
       http://www.springframework.org/schema/context
       http://www.springframework.org/schema/context/spring-context.xsd">
    <context:component-scan base-package="com.javaee.ex08.controller"/>
<!--    <bean class="org.springframework.web.servlet.handler.SimpleMappingExceptionResolver">
        <property name="exceptionMappings">
            <props>
                <prop key="java.lang.ArithmeticException">exception/arithmetic_exception.jsp</prop>
                <prop key="java.lang.ArrayIndexOutOfBoundsException">exception/out_of_bounds_exception.jsp</prop>
                <prop key="MyException">exception/self_defined_exception.jsp</prop>
            </props>
        </property>
        <property name="defaultErrorView" value="exception/else_exception.jsp"/>
        <property name="exceptionAttribute" value="exp_info"/>
    </bean>-->
</beans>
